### DISCUSSION TITLE
Google authentication with supabase iOS Swift application

### DISCUSSION THREAD

**Discussion by HuongCao** [supabase/supabase#5793] [Questions]
I'm following the readme at https://github.com/supabase-community/gotrue-swift

- signIn with oauth2 provider only return provider url. Users have to launch that url to continue the auth flow.
- After receiving callback uri from oauth2 provider, use getSessionFromUrl to parse session data.

But I want to know how to get the callback uri from oauth2 provider on iOS app?

Below is the code I'm using the google authentication with supabase Swift.
![Screen Shot 2022-03-04 at 17 39 54](https://user-images.githubusercontent.com/17448131/156749251-a0e1dbb0-c332-4974-addd-947438a8f69d.png)
Currently, I saw the URL like thishttps://[MY_APP_ID].supabase.co/auth/v1/authorize?provider=google

Please help to advise!
Thanks!

**Comment by OceanTechnic** [comment]
@HuongCao Have you solved this? I am also trying to solve similar problem

**Reply by hhopkins95** [reply]
Have either of you solved this? I'm having the same issue

**Comment by tonygaglio** [comment]
@HuongCao Checking in on this - has this been addressed?
